.github/workflows/build.yml

```diff
@@ -94,11 +95,26 @@ jobs:
             7z a chatterino-windows-x86-64.zip Chatterino2/
         shell: cmd
 
+      - name: Build with CMake (Windows)
+        if: startsWith(matrix.os, 'windows') && matrix.build-system == 'cmake'
+        run: |
+            call "%programfiles(x86)%\Microsoft Visual Studio\2019\Enterprise\VC\Auxiliary\Build\vcvars64.bat"
+            mkdir build
+            cd build
+            "C:\Program Files\Conan\conan\conan.exe" install ..\conanfile-qmake.txt
+            cmake ..
+            nmake
```

Comment:
nmake doesn't work with sln projects, you have to use ether `msbuild /p:Configuration=Release /m Chatterino2.sln` or use `cmake -G"NMake Makefiles" -DCMAKE_BUILD_TYPE=Release ..`